scripts/pivotal-listener.coffee

```diff
@@ -43,7 +43,8 @@ module.exports = (robot) ->
         "DAR - ENG": "DARENG-",
         "Salesforce": "SFDC-",
         "JAM": "JAM-",
-        "Business As Usual": "BAU-"
+        "Business As Usual": "BAU-",
+        "State Of The States 2019": "SOTS-"
```

Comment:
@mhurty do you mind that we are naming stories on two boards `SOTS-` ? I don't _think_ pivotal will have an issue with duplicate names for cards since they track unique IDs.

Reply:
maybe that's that problem. I could update the name appendage to SOTS19- to avoid the duplication.